nebula-exchange/src/main/scala/com/vesoft/nebula/exchange/processor/VerticesProcessor.scala

```diff
@@ -93,6 +94,7 @@ class VerticesProcessor(data: DataFrame,
         s"${config.errorConfig.errorPath}/${tagConfig.name}.${TaskContext.getPartitionId()}")
       errorBuffer.clear()
     }
+    LOG.info(s"ver part-costTime>>>>>>>>>>>>>>>>>${TaskContext.getPartitionId()}-${System.currentTimeMillis() - startTime}")
```

Comment:
Ditto, is it better to change `ver part` to `spark partition for vertex`?

Reply:
already change it